src/cdk-experimental/popover-edit/popover-edit.spec.ts

```diff
@@ -750,3 +850,7 @@ function getCells(row: Element): HTMLElement[] {
 
   return getElements(row, '.cdk-cell, td');
 }
+
+function clearLeftoverTimers() {
```

Comment:
Add a comment somewhere in this file that explains why you expect there to be leftover timers?